editor/syncclient.cpp

```diff
@@ -107,6 +107,14 @@ void AbstractSocketClient::processCommand()
 		case SET_ROW:
 			processSetRow();
 			break;
+
+		case SET_KEY:
+			processSetKey();
+			break;
+
+		case PAUSE:
+			processPause();
```

Comment:
I would emit the signal here rather than hiding it behind a wrapper, since it's so trivial.